ios/RNSnackBarView.m

```diff
@@ -262,7 +262,8 @@ - (void)show {
 
     id textColor = _pendingOptions[@"textColor"];
     self.textColor = textColor ? [RCTConvert UIColor:textColor] : [UIColor whiteColor];
-
+    textLabel.font = [UIFont fontWithName:_pendingOptions[@"fontFamily"] size:15];
```

Comment:
Clarifying question: do you know that `15` is the default size? Why does this need to be explicitly set?

Reply:
@mmmelsayed

Reply:
@cooperka Hi, whats the status of the PR ? if the author doesn't respond I can open another PR related to this. Our current project have an issue related the text fon changing in iOS